package.json

```diff
@@ -125,6 +125,10 @@
     "webpack-dev-server": "^5.1.0",
     "yarn": "^1.22.18"
   },
+  "engines": {
+    "node": ">=18.20.5",
```

Comment:
v18 is effectively E0F - perhaps we should consider using v20 as minimum?
https://nodejs.org/en/about/previous-releases